### ISSUE TITLE
[RFC] bottom border on sign-up form

### ISSUE THREAD

**Issue by unarist** [MasterDalK/kktCSS#76]
### Remove border and radius on the bottom

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/705555/29164649-22389de6-7dfb-11e7-896e-4075e14c9f61.png)|![1502343955101](https://user-images.githubusercontent.com/705555/29164405-43d84e02-7dfa-11e7-834e-91bf7dc9e078.jpg)|

### Remove and add full width border

![image](https://user-images.githubusercontent.com/705555/29164463-69a35ec4-7dfa-11e7-8d31-d1967ad0d6d7.png)

### Remove and add full width shadow

![image](https://user-images.githubusercontent.com/705555/29164512-9776ac7a-7dfa-11e7-877e-e576e66b6936.png)

This shadow looks not bad, but I'm not sure whether it is correct in lighting.

**Comment by MasterDalK** [comment]
I like the shadow with removed border. We'll probably need to keep the border for mobile though

**Comment by unarist** [comment]
shadow example:

```css
.landing-page .header-wrapper .mascot {
    z-index: 5;
}
.header-wrapper:after {
    content: "";
    display: block;
    position: absolute;
    bottom: -10px;
    height: 10px;
    width: 100%;
    box-shadow: 0 -2px 2px rgba(0,0,0,0.1);
    z-index: 4;
}
```

**Comment by unarist** [comment]
fixed at v10.0 #75
